receiver/stdinreceiver/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/stdinreceiver
+
+go 1.14
+
+require (
+	github.com/gorilla/mux v1.8.0
+	github.com/stretchr/testify v1.6.1
+	go.opencensus.io v0.22.5
```

Comment:
I don't see where opencensus is needed. Same for mux. Please check and remove if not needed.

Reply:
Good catch, removed.